telegram/_bot.py

```diff
@@ -3665,7 +3665,7 @@ async def edit_message_text(
         if inline_message_id:
             data["inline_message_id"] = inline_message_id
         if entities:
-            data["entities"] = [me.to_dict() for me in entities]
+            data["entities"] = [me.to_dict(recursive=True) for me in entities]
```

Comment:
Are the changes in `_bot` necessary?

Reply:
no, I passed in the parameter cause you like it 😃